blackduck/Client.py

```diff
@@ -56,33 +56,41 @@ def __init__(self, base_url, timeout, retries, verify):
 
     def request(self, method, url, **kwargs):
         kwargs['timeout'] = self._timeout
+        if method.lower() == 'get':
+            # set default media type if not provided
+            headers = {
+                'accept': "application/json",
+                'content-type': "application/json"
```

Comment:
Is content-type not also used by post? what is the benefit of performing this check?

Reply:
Good point. I'll test out post today/tomorrow and adjust as needed.